go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/mattermost/mattermost-plugin-api v0.0.22-0.20211210183909-beb4761e4bd3
 	github.com/mattermost/mattermost-server/v6 v6.0.0-20220315170027-7710a2fe3741
-	github.com/nicksnyder/go-i18n/v2 v2.1.2
+	github.com/nicksnyder/go-i18n/v2 v2.2.0
```

Comment:
Would you please also update https://github.com/mattermost/mattermost-plugin-apps/blob/c14f3d4a76de3b8801e456cd83be56b59ce78746/.circleci/config.yml#L95?